tests/unit/BusinessTime/AddBusinessDays.test.ts

```diff
@@ -51,6 +57,15 @@ describe("adding business days", () => {
         ["Friday 2018-05-18 17:00", 1.75, "Tuesday 2018-05-22 15:00"],
         ["Friday 2018-05-18 17:00", 2, "Tuesday 2018-05-22 17:00"],
         ["Friday 2018-05-18 17:00", 3, "Wednesday 2018-05-23 17:00"],
+
+        // From Friday morning
+        ["Tuesday 2021-10-19 00:00", 2, "Thursday 2021-10-21 00:00"],
+        ["Friday 2021-10-29 09:01", 1, "Monday 2021-11-01 09:01"],
+        ["Friday 2021-10-29 10:00", 1, "Monday 2021-11-01 10:00"],
+        ["Friday 2021-10-29 04:00", 1, "Monday 2021-11-01 04:00"],
+        ["Friday 2021-10-29 00:00", 1, "Monday 2021-11-01 00:00"],
```

Comment:
Maybe worth adding a new case to the negative values below for Monday -1 = Friday? I think it might have issues where before we called `this.add(days, 'days')` and `days` could be negative, but now we always increment by `1`

Reply:
Yeah, especially because the subtraction is not yet implemented :D

Reply:
added to tests/unit/BusinessTime/SubtractBusinessDays.test.ts